### ISSUE TITLE
More messages on custom template with blockui directive

### ISSUE THREAD

**Issue by ddorigo** [kuuurt13/ng-block-ui#29]
In the custom template configuration I have two messages to be updated.
 One that describes the current progress and another with the percentage number. 
Currently I can update only a message string. 
I tried also to get access to the custom component; but without success.

**Comment by kuuurt13** [comment]
@ddorigo I think I am following your issue. Currently I don't think this would be possible within the library since you can only pass a single string to the message like you said. Let me look into this and try to get back to you with a possible solution to allow for this scenario when using the library.

**Comment by ddorigo** [comment]
The only solution I found was to pass a number as message and compare in the template div sections using *ngIf I set different messages. It works but cumbersome.

**Comment by kuuurt13** [comment]
So I looked into this and I think the fix would be to pass an object as the message so in your case message would be `{ progress, percentage }` and then in the view you would access them. I created a branch of the library that changes message from `string` to `any` to allow for this. I created this [plunker](https://embed.plnkr.co/ACBGxgoWveb62r75nvyB?show=block-template.component.ts,preview&autoCloseSidebar) to go along with it. Take a look and let me know if this would fix your issue.
